src/test/java/com/baidu/hugegraph/unit/util/UnitUtilTest.java

```diff
@@ -0,0 +1,667 @@
+        Assert.assertEquals("8.75 PB", value);
+
+        value = UnitUtil.bytesToReadableString(Bytes.PB * 168 +
+                                               (long) (756 * 1.024) * Bytes.TB);
+        Assert.assertEquals("168.76 PB", value);
+
+        // EB
+        value = UnitUtil.bytesToReadableString(Bytes.PB * 1024);
+        Assert.assertEquals("1 EB", value);
+
+        value = UnitUtil.bytesToReadableString(Bytes.EB + 1 * Bytes.PB);
+        Assert.assertEquals("1.0 EB", value);
+
+        value = UnitUtil.bytesToReadableString(Bytes.EB + 10 * Bytes.PB);
+        Assert.assertEquals("1.01 EB", value);
+
+        value = UnitUtil.bytesToReadableString(Bytes.EB + 20 * Bytes.PB);
+        Assert.assertEquals("1.02 EB", value);
+
+        value = UnitUtil.bytesToReadableString(Bytes.EB + 100 * Bytes.PB);
+        Assert.assertEquals("1.1 EB", value);
+
+        value = UnitUtil.bytesToReadableString(Bytes.EB + 123 * Bytes.PB);
+        Assert.assertEquals("1.12 EB", value);
+
+        value = UnitUtil.bytesToReadableString(Bytes.EB + 129 * Bytes.PB);
+        Assert.assertEquals("1.13 EB", value);
+
+        value = UnitUtil.bytesToReadableString(Bytes.EB * 7 +
+                                               (long) (755 * 1.024) * Bytes.PB);
+        Assert.assertEquals("7.75 EB", value);
+
+        // Bytes overflow long value
+        value = UnitUtil.bytesToReadableString(Bytes.EB * 8);
+        Assert.assertEquals("0 B", value);
+    }
+
+    @Test
+    public void testBytesFromReadableString() {
+        // B
+        Assert.assertEquals(0L, UnitUtil.bytesFromReadableString("0 B"));
+        Assert.assertEquals(1L, UnitUtil.bytesFromReadableString("1 Bytes"));
+        Assert.assertEquals(3L, UnitUtil.bytesFromReadableString("3 bytes"));
+        Assert.assertEquals(10L, UnitUtil.bytesFromReadableString("10 Byte"));
+        Assert.assertEquals(100L, UnitUtil.bytesFromReadableString("100 byte"));
+        Assert.assertEquals(1000L, UnitUtil.bytesFromReadableString("1000 b"));
+        Assert.assertEquals(1023L, UnitUtil.bytesFromReadableString("1023 B"));
+
+        Assert.assertEquals(1024L, UnitUtil.bytesFromReadableString("1024 B"));
+        Assert.assertEquals(10245678L,
+                            UnitUtil.bytesFromReadableString("10245678 B"));
+        Assert.assertEquals(102456789012L,
+                            UnitUtil.bytesFromReadableString("102456789012 B"));
+
+        Assert.assertEquals(1L, UnitUtil.bytesFromReadableString("1  B"));
+        Assert.assertEquals(1L, UnitUtil.bytesFromReadableString("1  B "));
+        Assert.assertEquals(-2L, UnitUtil.bytesFromReadableString("-2 B"));
+
+        // KB
+        Assert.assertEquals(0L, UnitUtil.bytesFromReadableString("0 KB"));
+        Assert.assertEquals(Bytes.KB * 1L,
+                            UnitUtil.bytesFromReadableString("1 KB"));
+        Assert.assertEquals((long) (Bytes.KB * 3.14),
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [CNT: Rough value of known constant found (CNT_ROUGH_CONSTANT_VALUE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#cnt-rough-value-of-known-constant-found-cnt-rough-constant-value).
It's recommended to use the predefined library constant for code clarity and better precision.